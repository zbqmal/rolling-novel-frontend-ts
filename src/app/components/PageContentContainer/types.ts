export type PageContentContainerProps = React.PropsWithChildren<{
  width?: string;
  height?: string;
}>;
