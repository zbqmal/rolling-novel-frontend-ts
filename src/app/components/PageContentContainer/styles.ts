export const pageContentContainerStyles = {
  container: {
    width: "100%",
    height: "100%",
    backgroundColor: "white",
    display: "flex",
    flexWrap: "wrap" as const,
    gap: "10px",
  },
};
