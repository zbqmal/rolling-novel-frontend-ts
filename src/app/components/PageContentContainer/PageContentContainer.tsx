import React from "react";
import { pageContentContainerStyles } from "./styles";
import { PageContentContainerProps } from "./types";

const PageContentContainer: React.FC<PageContentContainerProps> = ({
  width,
  height,
  children,
}) => {
  return <div style={pageContentContainerStyles.container}>{children}</div>;
};

export default PageContentContainer;
