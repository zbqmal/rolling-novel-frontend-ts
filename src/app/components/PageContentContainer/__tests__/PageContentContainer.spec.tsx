import { render } from "@testing-library/react";
import PageContentContainer from "../PageContentContainer";

describe("PageContentContainer", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<PageContentContainer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
