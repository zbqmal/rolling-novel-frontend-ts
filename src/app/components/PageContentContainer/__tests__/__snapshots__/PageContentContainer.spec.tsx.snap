// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContentContainer matches snapshot 1`] = `
<DocumentFragment>
  <div
    style="width: 100%; height: 100%; background-color: white; display: flex; flex-wrap: wrap; gap: 10px;"
  />
</DocumentFragment>
`;
